libs/clients/health-directorate/src/lib/clients/health/health.service.ts

```diff
@@ -0,0 +1,151 @@
+import { Auth, AuthMiddleware } from '@island.is/auth-nest-tools'
+import { handle404 } from '@island.is/clients/middlewares'
+import type { Logger } from '@island.is/logging'
+import { LOGGER_PROVIDER } from '@island.is/logging'
+import { Inject, Injectable } from '@nestjs/common'
+import {
+  DispensationDto,
+  Locale,
+  PrescriptionsApi,
+  ReferralsApi,
+  WaitingListsApi,
+  PrescribedItemDto,
+  ReferralDto,
+  WaitingListEntryDto,
+  DispensationHistoryDto,
+} from './gen/fetch'
+
+const LOG_CATEGORY = 'health-directorate-health-api'
+
+@Injectable()
+export class HealthDirectorateHealthService {
+  constructor(
+    @Inject(LOGGER_PROVIDER) private readonly logger: Logger,
+    private readonly referralsApi: ReferralsApi,
+    private readonly waitingListsApi: WaitingListsApi,
+    private readonly prescriptionsApi: PrescriptionsApi,
+  ) {}
+
+  private referralsApiWithAuth(auth: Auth) {
+    return this.referralsApi.withMiddleware(new AuthMiddleware(auth))
+  }
+
+  private waitingListsApiWithAuth(auth: Auth) {
+    return this.waitingListsApi.withMiddleware(new AuthMiddleware(auth))
+  }
+
+  private prescriptionsApiWithAuth(auth: Auth) {
+    return this.prescriptionsApi.withMiddleware(new AuthMiddleware(auth))
+  }
+
+  private mapLocale(locale: string): Locale {
+    return locale === 'is' ? Locale.Is : Locale.En
+  }
+
+  /* Afgreiðslur */
+  public async getDispensations(
+    auth: Auth,
+    atcCode: string,
+    locale: string,
+  ): Promise<Array<DispensationHistoryDto> | null> {
+    const dispensations = await this.prescriptionsApiWithAuth(auth)
+      .mePrescriptionDispensationControllerGetDispensationsForAtcCodeV1({
+        atcCode,
+        locale: this.mapLocale(locale),
+      })
+      .catch(handle404)
+
+    if (!dispensations) {
+      this.logger.debug(
+        `No prescription dispensations returned for atc code: ${atcCode}`,
+        {
+          category: LOG_CATEGORY,
+        },
```

Comment:
```suggestion
        `No prescription dispensations returned for atc code`,
        {
          atcCode,
          category: LOG_CATEGORY,
        },
```

now we can filter by `atcCode` in datadog

Reply:
Yes! Good idea ✨